deluge/ui/gtkui/statusbar.py

```diff
@@ -326,6 +331,12 @@ def _on_max_upload_speed(self, max_upload_speed):
         self.max_upload_speed = max_upload_speed
         self.update_upload_label()
 
+    def _on_get_external_ip(self, external_ip):
+        if external_ip:
```

Comment:
Good point, at the time I thought it could be neater...

I have been making the mistake of thinking I should avoid `%s` in the translation string but found since then that [concat is not correct](http://read-the-docs.readthedocs.io/en/latest/i18n.html#interpolation). The string should use substitution (or format) so that translators can change the position of `%s` if the word order is different. With this new knowledge I may have to go back through the code and make the correct alterations!